src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const mainPhoto = document.querySelector('#largeImg');
+const imagesContainers = document.querySelectorAll('.list-item');
+
+imagesContainers.forEach((el) => {
+  el.addEventListener('click', (e) => {
```

Comment:
The current implementation attaches an individual click event listener to each .list-item, which can be inefficient if there are many such elements. Using event delegation by attaching a single event listener to a common ancestor would improve performance and simplify management, especially if .list-item elements are dynamically added or removed. [[more about event delegation]](https://www.freecodecamp.org/news/event-delegation-javascript/)